app/src/main/java/com/halim/adam/cashmaster/AddNewBudget.java

```diff
@@ -0,0 +1,24 @@
+package com.halim.adam.cashmaster;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+
+public class AddNewBudget extends Activity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_add_new_budget);
+    }
+
+    public void GetInput(View view){
+        EditText inputName = findViewById(R.id.inputName);
+        EditText inputPortion = findViewById(R.id.inputPortion);
+        EditText inputAmount = findViewById(R.id.inputAmount);
+
```

Comment:
Feedback: It would be nice if there's a validation on the inputAmount, because if nothing is inputted, this will crash on line 22